### DISCUSSION TITLE
HB install

### DISCUSSION THREAD

**Discussion by KDRFLT** [humbertogontijo/homeassistant-roborock#145] [General]
Hello,

Is there any information on how to install this on HomeBridge?

thx

**Comment by shred86** [comment]
This is an integration for Home Assistant so it does not work with HomeBridge. 

HomeKit also doesn’t have a vacuum entity so if someone did create a HomeBridge plugin, it would have to be integrated as something like a fan or a bunch of switches. I honestly don’t think it would be very useful. 

One thing I do is create virtual switches in Home Assistant and expose them to HomeKit to perform specific functions, like “Vacuum the kitchen”, so I can use it with Siri.
